scripts/validate-migration-snapshots.ts

```
@@ -0,0 +1,267 @@
+		dialect: parsed.data.dialect,
+	};
+}
+
+export function validatePrevIdChain(snapshots: SnapshotInfo[]): string[] {
+	const errors: string[] = [];
+
+	if (snapshots.length === 0) {
+		return errors;
+	}
+
+	const first = snapshots[0]!;
```

Biome (2.4.9)

[error] Forbidden non-null assertion.

(lint/style/noNonNullAssertion)